maven-plugin/src/main/java/org/moditect/mavenplugin/add/AddModuleInfoMojo.java

```diff
@@ -1,399 +0,0 @@
-/**
```

Comment:
It's a bit unfortunate that the relocation and changes where done in a single commit. This makes it nearly impossible to grok what has changed within the code. It'd be very helpful to split up the commit into multiple ones, e.g. one to move classes as needed without actually applying code changes to them, and then do these changes in separate commits.